launch/testnet/server.go

```diff
@@ -61,6 +63,18 @@ func Setup() *Server {
 		}
 	}
 
+	// Instantiate GraphQL server
+	if cfg.Get().Gql.Enabled {
+		gqlServer, err := gql.NewHTTPServer(eventBus, rpcBus)
+		if err != nil {
+			log.Errorf("graphQL server error: %s", err.Error())
+		}
+
+		if err := gqlServer.Start(); err != nil {
+			log.Errorf("graphQL server error: %s", err.Error())
```

Comment:
Would it be better to have distinct error messages for these two function calls? Instead of the current one, we could do something like: `error creating graphQL server` and `error starting graphQL server`, to make it more clear where the error is coming from.

Reply:
Good catch!